lib/edge.js

```diff
@@ -92,11 +92,17 @@ class Edge {
       type: this.type,
       name: this.name,
       spec: this.spec,
+      bundled: this.bundled,
```

Comment:
This gets quite noisy.

I recommend changing the line above to `const { error, from, bundled } = this`, and then change this line to

```js
  ...(bundled ? { bundled } : {}),
```

so it's filtered out like `from` and `error` when not relevant.

Reply:
or
```
...(bundled && { bundled }),
```

Reply:
i feel more sympathy for v8 wanting consistent object shape than noise in the snapshots, but can do

Reply:
did this